brax/envs/base.py

```diff
@@ -139,10 +140,14 @@ def dt(self) -> jax.Array:
     return self.sys.opt.timestep * self._n_frames  # pytype: disable=attribute-error
 
   @property
-  def observation_size(self) -> int:
+  def observation_size(self) -> ObservationSize:
     rng = jax.random.PRNGKey(0)
     reset_state = self.unwrapped.reset(rng)
-    return reset_state.obs.shape[-1]
+    obs = reset_state.obs
+    if isinstance(obs, jax.Array) and len(obs.shape) == 1:
```

Comment:
why do we need the len(obs.shape) == 1 ? just do what we had on the left hand side

Reply:
Added clarifying comment. Does it make sense to have observation_size return a tuple for multi-dimensional obs?

Reply:
Please fix